src/components/mediainfo/mediainfo.js

```diff
@@ -257,11 +257,18 @@ import '../../elements/emby-button/emby-button';
             if (item.Type === 'Audio') {
                 miscInfo.push(datetime.getDisplayRunningTime(item.RunTimeTicks));
             } else {
-                minutes = item.RunTimeTicks / 600000000;
```

Comment:
Could we just use `getDisplayRunningTime()` here also? It would display in the `00:00:00` format.